### DISCUSSION TITLE
[Suggestion] Elytra compatibility with Customizable Player Models mod

### DISCUSSION THREAD

**Discussion by kotmatross28729** [Roadhog360/Et-Futurum-Requiem#441] [Ideas]
There is a mod called [Customizable Player Models](https://github.com/tom5454/CustomPlayerModels), which allows you to change your model, skin, animations, etc. One of the things in this mod is that you can set a custom texture for the cape/elytra. Texture for the cape works, but if you set the texture for the elytra, it will not be displayed on the Et futurum elytra. My suggestion is to add Customizable Player Models Elytra skin compatibility for Et futurum Elytra. By the way, there is a similar implementation is in the [OfflineAuth mod](https://github.com/trollogyadherent/OfflineAuth). Although I’m not exactly sure from which side this needs to be implemented

![2024-03-24_11 10 34](https://github.com/Roadhog360/Et-Futurum-Requiem/assets/110309314/808d7847-70d0-4680-b8b2-3943f9f45837)

![2024-03-24_11 10 43](https://github.com/Roadhog360/Et-Futurum-Requiem/assets/110309314/89ff9b8e-450a-4471-ba87-285c06cbb240)

**Comment by Roadhog360** [comment]
The mod is likely not using vanilla's cape system, hence why it isn't being picked up. I'm unsure how I'd detect the More Player Models cape.

**Comment by Roadhog360** [comment]
Closing, this is a bug on their end; please make a request on their issue tracker for them to use vanilla's cape URLs. If they have a problem with this, you can point them to this discussion here and I will discuss with them
